core/classes/FilterConverter.class.php

```diff
@@ -614,6 +614,7 @@ private function convertTypeToJson( &$p_criteria ) {
 		unset( $p_criteria['_version'] );
 		unset( $p_criteria['_source_query_id'] );
 		unset( $p_criteria['_view_type'] );
+		unset( $p_criteria['_filter_id'] );
```

Comment:
The code here is to remove `_filter_id` from the criteria, since it is usually surfaced as `id` and it is part of the filter rather than the filter criteria.